plugins/woocommerce/tests/e2e-pw/tests/merchant/customer-list.spec.js

```diff
@@ -114,9 +114,14 @@ test.describe( 'Merchant > Customer List', () => {
 					.fill( customer.first_name );
 				await page
 					.getByRole( 'option', {
-						name: 'All customers with names that',
+						name: `All customers with names that include ${ customer.first_name }`,
 					} )
-					.click();
+					.waitFor( { state: 'visible' } );
+				await page
+					.getByRole( 'option', {
+						name: `All customers with names that include ${ customer.first_name }`,
+					} )
+					.click( { delay: 300 } );
```

Comment:
Wish we didn't have to keep hardcoding these delays in.  Playwright is supposed to wait for it to be visible and clickable automatically -- is there an XHR request going on behind the scenes here that's causing an issue?

Reply:
Yeah, I don't like it much, but it seems like the easiest and fastest solution. Sometimes, PW is just too fast in repetitive steps like this, and it then depends on how WooCommerce will accept inhuman reactions.